megazords/ios-rust/MozillaTestServices/MozillaTestServices.xcodeproj/project.pbxproj

```diff
@@ -68,7 +68,6 @@
 		39F5D766295616E3004E2384 /* NimbusBuilder.swift in Sources */ = {isa = PBXBuildFile; fileRef = 39F5D765295616E3004E2384 /* NimbusBuilder.swift */; };
 		45CC574A28AD9C86006D55AA /* errorsupport.udl in Sources */ = {isa = PBXBuildFile; fileRef = 45CC574828AD9C31006D55AA /* errorsupport.udl */; };
 		AB9C392E2A0DA61900AF5ADE /* remote_settings.udl in Sources */ = {isa = PBXBuildFile; fileRef = AB65933B2A0C524E00DBF059 /* remote_settings.udl */; };
-		BCFF93E82AA7E1ED005B5B71 /* suggest.udl in Sources */ = {isa = PBXBuildFile; fileRef = 3082A3E24B1A68B03DB4EEA3 /* suggest.udl */; };
```

Comment:
Since we don't have a `suggest.udl` file anymore, I am not sure how this has to change. Maybe @linabutler knows?

Reply:
Lina definitely knows best, but remove the UDL file from the sources seems right to me.  AFAIK, the only reason it's there is to trigger a recompile if it changes.

Reply:
This looks totally fine to me—I don't think we have any Swift-specific tests, so I wonder if we can remove the entire `suggest` group (including the `Generated/` subfolder), in addition to the UDL?

Reply:
Hm, I might need a direct pointer to that! Do you mean this part? https://github.com/mozilla/application-services/blob/main/megazords/ios-rust/MozillaTestServices/MozillaTestServices.xcodeproj/project.pbxproj#L669-L677

Reply:
Yep!

Let's try deleting lines 71 (✅), 190 (✅), 356, 571-579, 669-677, 814 (✅).

I wish GitHub could link to non-contiguous ranges.